guidelines/index.html

```diff
@@ -372,11 +372,11 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                   					
                </li>
                					
-               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">The WCAG 2 Documents</a></strong> - A brief introduction to the WCAG 2 supporting documents and supplemental guidance.</p>
+               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">The WCAG 2 Documents</a></strong> - WCAG 2 の関連文書及び補足的なガイダンスの簡単な紹介。</p>
                   					
                </li>
                
-               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> introduces the new success criteria with persona quotes that illustrate the accessibility issues.</p>
+               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> では、アクセシビリティの課題を示すペルソナの引用を交えて、新しい達成基準を紹介している。</p>
```

Comment:
辞書寄りの翻訳にした以下はどうでしょうか。

```suggestion
               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> では、アクセシビリティの問題を説明するペルソナの引用とともに、新しい達成基準を紹介している。</p>
```

Reply:
違和感ありません。イイと思います！